bpf/lib/nodeport.h

```diff
@@ -1254,6 +1257,8 @@ static __always_inline int nodeport_lb4(struct __ctx_buff *ctx,
 	if (IS_ERR(ret)) {
 		if (ret == DROP_UNKNOWN_L4)
 			return CTX_ACT_OK;
+		else if (ret == DROP_NO_SERVICE)
+			goto skip_service_lookup;
```

Comment:
I would reorder with DROP_UNKNOWN_L4 since it's more likely to hit DROP_NO_SERVICE, imho than unknown L4 proto.